app/Http/Controllers/FaqController.php

```
@@ -0,0 +1,20 @@
+namespace App\Http\Controllers;
+
+use App\Actions\SeoSetFaqsPageAction;
+use App\Models\Faq;
+use Illuminate\Http\Request;
+use Illuminate\View\View;
+
+class FaqController extends Controller
+{
+    public function __invoke(Request $request, SeoSetFaqsPageAction $seoSetFaqsPageAction): View
+    {
+        $faqs = Faq::all();
```

PHPMD (2.15.0)

Avoid using static access to class '\App\Models\Faq' in method '__invoke'. (Clean Code Rules)

(StaticAccess)